smart-contracts/scripts/unlock-upgrade.js

```diff
@@ -0,0 +1,37 @@
+const { ethers, upgrades } = require('hardhat')
+const OZ_SDK_EXPORT = require('../openzeppelin-cli-export.json')
+
+const { getNetworkName } = require('../helpers/network')
+
+async function main() {
+  const chainId = await web3.eth.net.getId()
+  const networkName = process.env.RUN_MAINNET_FORK
+    ? 'mainnet'
+    : getNetworkName(chainId)
+
+  // eslint-disable-next-line no-console
+  console.log(`Deploying new implemntation on ${networkName}...`)
+
+  const [UDTInfo] =
+    OZ_SDK_EXPORT.networks[networkName].proxies['unlock-protocol/Unlock']
```

Comment:
Do u thin we could make this a cli param?

Reply:
ok, much simpler :)

```
npx hardhat upgrade --contract contracts/Unlock.sol --network localhost
```